tests/functional/app.js

```diff
@@ -5,10 +5,13 @@ const { itemsService } = require('./resources/item');
 const { errorsService } = require('./resources/error');
 const { headersService } = require('./resources/headers');
 const { slowService } = require('./resources/slow');
+const { slowFastService } = require('./resources/slowFast');
```

Comment:
slow fast?

Reply:
Hmmm, I should document/name it better. But the idea of this one is to give 2 slow responses and then a fast one. In this way it's possible to force the client to cancel the first 2 requests due to timeout but still get a good response in the 3rd try. Here, we are making sure that we are replacing the AbortController correctly between retries (instead of using the same one that would trigger an already aborted fetch call).